### PR TITLE
update to 0.21.1 and add unittests

### PR DESCRIPTION
**Pull request by dwerder**
#### What this PR does / why we need it:

  - updates chart to support flux 0.21.1
    - add new labels like in flux upstream manifestgen: https://github.com/fluxcd/flux2/pull/2036
    - make eventsaddr configurable to math https://github.com/fluxcd/flux2/pull/2046
    - match https://github.com/fluxcd/flux2/pull/2051
  - adds basic unittests
  - add bug and feature report templates for github

#### Special notes for your reviewer:

This is a rather big change. Sorry for that. You can ignore the files in \_\_snapshot\_\_, because these are autogenerate files by by helm unittest.

#### Checklist

- [x] [DCO](https://github.com/fluxcd-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] helm-docs are updated
- [ ] Helm chart is tested

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional-ctxt
+    attributes:
+      label: Additional context.
+      description: Add any other context or screenshots about the feature request here.
+    validations:
+      required: false
```
Comment (haarchri) [inline .github/ISSUE_TEMPLATE/feature_request.yml:44]: please add new line

**Review by haarchri — COMMENTED** [review]
small nitpick - thanks for UnitTests and bump of upstream flux2 project

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (dwerder) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: done

**Comment by pierluigilenoci** [comment]
Thanks for this ❤️

**Pull request merged**